pypesto/optimize/optimizer.py

```diff
@@ -623,3 +629,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class ScipyDifferentialEvolutionOptimizer(Optimizer):
+    """
+    Global optimization using scipy's differential evolution optimizer.
+    Package homepage: https://scipy.org
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 1000}
+        self.options = options
+
+    @fix_decorator
+    @time_decorator
+    @history_decorator
+    def minimize(
+            self,
+            problem: Problem,
+            x0: np.ndarray,
+            id: str,
+            history_options: HistoryOptions = None,
+    ) -> OptimizerResult:
+        lb = problem.lb
+        ub = problem.ub
+        bounds = []
+        for iNumBounds in range(0, len(lb)):
+            bounds.append((lb[iNumBounds], ub[iNumBounds]))
+
+        if ScipyDifferentialEvolutionOptimizer is None:
+            raise ImportError(
+                "This optimizer requires an installation of scypi.")
```

Comment:
```suggestion
                "This optimizer requires an installation of scipy.")
```

Reply:
I deleted this if-statement (cf. Yanniks comment above)

Reply:
Done